src/main.cpp

```diff
@@ -174,6 +176,7 @@ const static struct option long_opt[] =
   {"pidfile",                     required_argument, NULL, PIDFILE},
   {"daemon",                      no_argument,       NULL, DAEMON},
   {"sas-use-signaling-interface", no_argument,       NULL, SAS_USE_SIGNALING_IF},
+  {"support-shared-ifcs",         no_argument,       NULL, SUPPORT_SHARED_IFCS},
```

Comment:
I'm slightly confused,  this is the config option, right?
Is this config option meant to use dashes or underscores to separate words?